homeassistant/components/samsungtv/media_player.py

```diff
@@ -307,7 +307,7 @@ async def _async_startup_dmr(self) -> None:
 
     async def _async_resubscribe_dmr(self) -> None:
         assert self._dmr_device
-        with contextlib.suppress(UpnpConnectionError):
+        with contextlib.suppress(UpnpConnectionError, UpnpResponseError):
```

Comment:
As an alternative, we could tag it in line 298, so we don't even attempt to resubscribe if it failed on original startup...

Reply:
This seems fine to do in a patch release.  Removing the resubscribe might break things though for other users.  

I guess we need to wait on https://github.com/home-assistant/core/issues/71833#issuecomment-1127310065 to determine which way to go with this.

Reply:
I have altered this to use try...except and log the exception instead of suppressing.

Also, the latest log shows `UpnpCommunicationError` and since both `UpnpConnectionError` and `UpnpResponseError` inherit `UpnpCommunicationError`, I have simplified to only handle the parent `UpnpCommunicationError` error.